app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/ModalSheetHeaderItem.kt

```diff
@@ -16,15 +17,18 @@ import com.wire.android.ui.theme.wireTypography
 @Composable
 fun ModalSheetHeaderItem(
     title: String? = null,
-    leadingIcon: @Composable () -> Unit = {}
+    leadingIcon: @Composable () -> Unit = {},
+    iconAction: () -> Unit = {},
```

Comment:
wouldn't the Row wrapping iconAction not always clickable even if we do not pass anything to iconAction and leadingIcon ? I think instead of passing an lambda iconAction() and leadingIcon() composable we could pass only one thingy like : leadingIcon : @Composable() -> Unit and then leave it up to consuming part if we want the Row to be clickable or not